doc/ModulePriority.md

```diff
@@ -0,0 +1,144 @@
+# How to configure priorities for modules
```

Comment:
Suggested text: "Module startup order priorities". 
@kgremban  to confirm

Reply:
"How to configure..." is good because having a verb in the title helps readers know what they'll be doing. But I agree with the later comment about being consistent around terminology, and startup order priority is more clear. So maybe "How to configure module startup order" or "How to configure startup order priority for modules"

Reply:
Going to change it as per your recommendation